content/02.main-text.md

```diff
@@ -98,13 +99,13 @@ The entire process can be orchestrated through GitHub with a web browser if a co
 We found that this workflow was an effective compromise between fully unrestricted editing and a more heavily-structured approach that limits the authors or the sections they can edit.
 In addition, authors are associated with their commits, which makes it easy for contributors to receive credit for their work and helps prevent ghostwriting [@doi:10.1371/journal.pmed.1000023].
 The GitHub contributors page summarizes all edits and commits from each author, providing aggregated information that is not available on other collaborative writing platforms.
-Because our writing process, like others backed by the open git version control system [@url:https://www.overleaf.com/; @url:https://www.authorea.com/], tracks the complete commit history, it also enables detailed retrospective contribution analysis.
+Because our writing process, like others backed by the open git version control system (including Overleaf and Authorea), tracks the complete commit history, it also enables detailed retrospective contribution analysis.
 `TODO: confirm Overleaf and Authorea provide this type of git integration versus something more coarse`
 
 ### Authorship
 
-To determine authorship we followed the International Committee of Medical Journal Editors (ICMJE) guidelines and used GitHub to track contributions.
-ICMJE recommends authors substantially contribute to, draft, approve, and agree to be accountable for the manuscript [@url:http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html].
+To determine authorship we followed the [International Committee of Medical Journal Editors (ICMJE) guidelines](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html) and used GitHub to track contributions.
```

Comment:
maybe just hyperlink `guidelines` as hyperlinking the entire thing will be visually prominent for a hyperlink of minimal importance.

Reply:
👍